contents/handbook/company/structure.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Small Teams
+sidebar: Handbook
+showTitle: true
+hideAnchor: true
+---
+
+PostHog is structured for speed, autonomy and innovation.
+
+Traditional organizations have big, separate functions. You have a product team, an engineering team, customer support, and so on. This slows things down, which stifles innovation - you have to get your boss to talk to someone else's boss to get work done. It also means that people can't really see the impact of their work.
```

Comment:
While I don't think it matters hugely in terms of the outcome we're trying to achieve here, this isn't really accurate. This is a _big_ oversimplification but, depending on size, you typically have companies organised by:

- Function
- Product (sometimes called 'business unit')
- Matrix (ie. combo of the above)

(Then you also have your wackier flavours like holocracy etc.)

I think the bigger problem with traditional orgs isn't which of these structures is 'best' but rather that growth in general leads to inefficiency due to a) more layers of communication and b) unnecessary 'approval' chains.